config/envs/default.py

```diff
@@ -32,9 +32,10 @@ class DefaultConfig:
     APPLICATION_ENDPOINT = "/applications/{account_id}"
 
     # Fund Store
-    FUND_STORE_API_HOST = environ.get("FUND_STORE_API_HOST")
+    FUND_STORE_API_HOST = environ.get("FUND_STORE_API_HOST", "fund_score_host")
```

Comment:
```suggestion
    FUND_STORE_API_HOST = environ.get("FUND_STORE_API_HOST", "fund_store_host")
```
Doesn't seem like this fallback is used at all so can it just be removed?